packages/apollo-cli/src/commands/file/upload.ts

```diff
@@ -0,0 +1,76 @@
+import { Flags } from '@oclif/core'
+import { Response } from 'undici'
+
+import { FileCommand } from '../../fileCommand.js'
+import { filterJsonList, queryApollo, wrapLines } from '../../utils.js'
+
+export default class Upload extends FileCommand {
+  static summary = 'Upload a local file to the Apollo server'
+  static description = wrapLines(
+    'This command only uploads a file and returns the corresponding file id.\
+    To add an assembly based on this file use `apollo assembly add-file`.\
+    To upload & add an assembly in a single pass see commands `apollo assembly add-*`',
+  )
+
+  static examples = [
+    {
+      description: wrapLines('Upload local file, type auto-detected:'),
+      command: '<%= config.bin %> <%= command.id %> -i genome.fa > file.json',
+    },
+  ]
+
+  static flags = {
+    'input-file': Flags.string({
+      char: 'i',
+      description: 'Local file to upload',
+      required: true,
+    }),
+    type: Flags.string({
+      char: 't',
+      description:
+        'File type or "autodetect" for automatic detection.\n\
+        NB: There is no check for whether the file complies to this type',
+      options: ['text/x-fasta', 'text/x-gff3', 'autodetect'],
```

Comment:
We need to add new options here, since we want to be able to tell the type later. Types I can think of are:
- text/x-fasta
- text/x-gff3
- application/x-bgzip-fasta
- text/x-fai
- application/x-gzi